Cinema Journal/Model/Networking.swift

```diff
@@ -26,9 +26,9 @@ struct Networking {
             guard let data = data, let response = response else { return }
             do {
                 let jsonData = try JSONDecoder().decode(T.self, from: data)
-                dump(jsonData)
+                completionHandler(jsonData as? [String : Any], nil)
```

Comment:
`completionHandler(jsonData, nil)

Reply:
Error: "_Cannot convert value of type 'T' to expected argument type '[String : Any]?_'"

Reply:
Firstly convert completionHandler at function signature like I pointed earlier

Reply:
Got it